lib/agents/weather/lambda.py

```diff
@@ -0,0 +1,140 @@
+from aws_lambda_powertools import Tracer
+from aws_lambda_powertools.event_handler import BedrockAgentResolver
+from aws_lambda_powertools.event_handler.openapi.params import Query
+from pydantic import BaseModel
+from typing import List
+import boto3
+import os
+import requests
+from typing import Annotated
+import pandas as pd
```

Comment:
nip: I believe pandas is large and would cause cold start due to numpy. I would remove this dependency if it's easy (I believe Panda has smaller managed lambda layers)